SU-SWS/saa_victoria_subtheme
# Adapt 696  email icon

# READY

# Summary
- Add support for color, placement of email icon
- hide other cta icons when both those and email icon are added onto a link

# Review By (Date)
- 9/16 asap

# Urgency
- medium/low but clients can see the issue on their websites right now. (infrequently used though)

# Steps to Test

1. Pull down the branch locally with monday's d8core update OR test on this page: https://saasierracamp-test.sites.stanford.edu/links-email and https://saasierracamp-test.sites.stanford.edu/  (this site has this branch on it for testing)
2. Clear the site cache
3. Test that simple url that goes to a mailto link includes an envelope icon
4. test that with any of the cta link options, the cta icon is hidden and only the envelope icon shows. it should NOT have both icons even if they try to add another icon, it should not look like this:
![image](https://user-images.githubusercontent.com/12898896/93269436-340cbc80-f76c-11ea-8920-1a99c3b8993b.png)

These work only when you add them as 'email' links in the email popup window when editing:
5. test that a mailto link works with correct styles and colors on red background for top red banner 
6. test that a mailto link works with correct styles and colors on red background forand in-page red banner (without an image in the in-page banner)


# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket ADAPT-696
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
